### PR TITLE
[Merged by Bors] - added page for `jolt` smartmodule

### PR DESCRIPTION
**Pull request by galibey**

### CODE REVIEW

File: content/smartmodules/certified/jolt.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+1 records outputed
+{"data":{"account":"1111","name":"John Smith","source":"http-connector","type":"custom"}}
+```
+
+%copy first-line%
+```shell
+$ smdk test --text '{"id":1, "name": "John Smith", "source":"mqtt-connector" }' --transforms-file ./transforms.yaml
+1 records outputed
+{"data":{"name":"John Smith","source":"mqtt-connector"}}
+```
```
Comment (ajhunyady) [inline content/smartmodules/certified/jolt.md:110]: For additional examples checkout the tutorials:
* data-pipeline
* mqtt-to-sql

**Review by ajhunyady — COMMENTED** [review]
Nice job! I suggest a few changes for better readability

File: content/smartmodules/certified/jolt.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: JSON to JSON Transformations
+menu: Jolt
+weight: 10
+---
+
+This is a certified by InfinyOn [map-type]({{<ref "../transform/map.md" >}}) SmartModule that transforms JSON records leveraging [Fluvio Jolt](https://github.com/infinyon/fluvio-jolt) library, which has its own DSL (Domain Specific Language) in order to facilitate its narrow job.
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The transformations in **Jolt** are a set of operations that are sequentially performed over an incoming record modifying that record.
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+-> Records in JSON text format are only supported
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+There are three main types of operations:
+* **Shift** - move the field from one location to another
+* **Default** - specify a default value for the field, if not existed
+* **Remove** - delete the field from the object
+
+There can be any amount of operations of any type in the transformation specification.
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+      "spec": {
+        "data": {
+            "source": "http-connector"
+        }
+      }
+    }
+]
+```
+
+The output of this transformation will be the result of operation `remove` followed by operation `shift` followed by operation `default`.
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+To demonstrate how **Jolt** SmartModule works we will use [SMDK]({{<ref "../smdk/build-test.md" >}}) tool. 
+
+First, we need to download it to our cluster:
+
+%copy first-line%
+```shell
+$ fluvio hub download infinyon/jolt@0.1.0
+```
+
+Second, we need to create a file `transform.yaml` with transformation specification that we already have:
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+%copy first-line%
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+            "*": "data.&0"
+        - operation: default
+          spec:
+            data:
+              source: "http-connector"
+ 
+
+```
+
+Now, we can test any records against our specification:
```
[the remark this answers is not in the document]
Comment (galibey) [reply content/smartmodules/certified/jolt.md]: done

File: content/smartmodules/certified/jolt.md (hunk 1 above)
Comment (galibey) [reply content/smartmodules/certified/jolt.md:110]: done

**Review by ajhunyady — APPROVED** [review]
Awesome!

**Pull request closed without merging**